src/components/DetailProductComments.tsx

```diff
@@ -1,25 +1,39 @@
 import { styled } from 'styled-components';
-import iconProfile from '/src/assets/ic_profile.svg';
-import imageEmpty from '/src/assets/Img_inquiry_empty.svg';
-import {
-  ContentText,
-  FlexWrapper,
-} from '/src/components/common/CommonComponents';
-import { Text } from '/src/components/common/CommonComponents';
-import Kebab from '/src/components/Kebab';
+import iconProfile from '@assets/images/ic_profile.svg';
+import imageEmpty from '@assets/images/Img_inquiry_empty.svg';
+import { ContentText, FlexWrapper } from '@components/common/CommonComponents';
+import { Text } from '@components/common/CommonComponents';
+import Kebab from '@components/Kebab';
+import { ReactElement } from 'react';
 
 const ProfileImage = styled.img`
   width: 40px;
   height: 40px;
 `;
 
-function Comment({ comment }) {
-  const { content, createdAt, updatedAt, writer } = comment;
+interface Writer {
+  image: string;
+  nickname: string;
+}
+
+interface Comment {
```

Comment:
```suggestion
interface IComment {
```
아래 Comment 컴포넌트와 이름이 겹칩니다!
이런걸 방지하기 위해서 interface는 I를, type은 T를 앞에 붙여주기도 합니다 (헝가리안표기법이라고합니다.)